### ISSUE TITLE
Rainbow Folders broken in Windows 1.6.2

### ISSUE THREAD

**Issue by kuyin** [mgmeyers/obsidian-style-settings#165]

**Comment by bofe126** [comment]
also broken in windows 1.6.3

**Comment by DaphneOdera17** [comment]
I got this problem too.

**Comment by KingCat-study** [comment]
I got this problem too.
in windows 1.6.3

**Comment by mgmeyers** [comment]
I am unsure what Rainbow Folders is, but it's not functionality that the Style Settings plugin provides. I recommend reporting any issues to whoever created Rainbow Files.

**Comment by bofe126** [comment]
Yes, it's Blue-Topaz, and they fixed this issue a couple of hours ago

**Comment by DaphneOdera17** [comment]
Update the Blue-Topaz theme can fix the problem.

**Comment by WardF** [comment]
I was using Rainbow Folders through the theme being managed by `Style Settings`; I had to go update the theme [as described here](https://github.com/AnubisNekhet/AnuPpuccin/tree/main?tab=readme-ov-file#updating), and rainbow folders started working again.

**Comment by sigrunixia** [comment]
resolved via theme update, not a Style Settings bug
